src/fairseq2/checkpoint.py

```diff
@@ -448,6 +461,8 @@ def raise_error(cause: Exception) -> NoReturn:
 
     @override
     def save_consolidated_fsdp_model(self, step_nr: int, model: Module) -> None:
+        log.info("Extracting consolidated FSDP state dictionary of the model.")
```

Comment:
nit: should this be a debug log instead of info log ?

Reply:
Hey @shagunsodhani, sorry just catching up with a pile of GitHub PRs and issues. This could be debug as well. The reason I kept it as info is because for large models (e.g. LLaMA 70B) this extraction step can take a while and it is helpful to know at what stage the checkpointing is while doing a regular training.

Reply:
Makes sense. Thank you for the clarification.